libportal/account.c

```diff
@@ -138,6 +138,12 @@ call_returned (GObject *object,
   ret = g_dbus_connection_call_finish (bus, result, &error);
   if (error)
     {
+      if (call->cancelled_id)
+        {
+          g_signal_handler_disconnect (g_task_get_cancellable (call->task), call->cancelled_id);
+          call->cancelled_id = 0;
+        }
+
```

Comment:
*Sigh* - I wish we could use `g_clear_signal_handler()` here, but it was introduced in GLib 2.62, which is 2 years old now. I'm not entirely sure we still want to an Ubuntu 18.04 CI job though.